packages/server/src/Store.ts

```diff
@@ -0,0 +1,7 @@
+export class Store {
+  rooms: string[];
```

Comment:
bot sure but maybe it's a good idea to use Set since duplicates can't be possible here
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set